Code/Engine/Foundation/Math/Implementation/Math_inl.h

```diff
@@ -80,6 +80,22 @@ namespace ezMath
 #endif
   }
 
+  EZ_ALWAYS_INLINE ezUInt32 FirstBitLow(ezUInt64 value)
```

Comment:
Didn't i just add such a function in my arr merge?

Reply:
Nope that was a different function